src/runtime.js

```diff
@@ -202,6 +216,31 @@ let tornado = {
   },
 
   /**
+   * Find and return a helper. If no helper of the given name is found, throw an error
+   * @param {String} name The name of the helper
+   * @param {Object} context The context at the point the helper was called
+   * @param {Object} params The params passed to the helper
+   * @return {DocumentFragment|Promise}
+   */
+  helper(name, context, params, bodies) {
+    let helper = this.helpers[name];
+    if (!helper) {
+      throw new Error(`Helper not registered: ${name}`);
+    } else {
+      return Promise.all(this.util.getValuesFromObject(params)).then(values => {
+        let resolvedParams = this.util.arraysToObject(Object.keys(params).sort(), values);
+        let returnVal = helper(context, resolvedParams, bodies);
+        if (!this.util.isPromise(returnVal)) {
+          returnVal = new Promise((resolve) => {
```

Comment:
Each use of `new Promise` in this file is different enough that a helper function wouldn't really... help.